docker/pypi/wmagent-base/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.cern.ch/cmsweb/oracle:21_5-stable as oracle
+FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230525
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+# Install basic OS package dependencies
+RUN apt-get update
+RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils cron mariadb-server myproxy voms-clients rlwrap libaio1 && apt-get clean
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps && apt-get clean
+
+# copy oracle client:
+COPY --from=oracle /usr/lib/oracle /usr/lib/oracle
+ENV LD_LIBRARY_PATH=/usr/lib/oracle
+ENV PATH=$PATH:/usr/lib/oracle
+ENV PKG_CONFIG_PATH=/usr/lib/oracle
+
+# Add all deployment needed directories, install and run scripts
+ADD etc /data/etc
```

Comment:
Are you absolutely sure that the contents in the `etc` you are adding here are coming from the correct source. Meaning from here: https://github.com/dmwm/CMSKubernetes/tree/master/docker/pypi/wmagent/etc 

If we are still copying only the Dockerfile in the CI pipeline I am not sure how could we add it without any error in this `wmagent-base` image. Have you inspected this in the image actually crated/uploaded to CERN registry?

Reply:
Yes, I did check the `wmagent-base` image and $WMA_ROOT_DIR/etc contains those 3 configuration files that have now been moved under the `wmagent-base` directory.